lib/saml/user_attributes/ssoe.rb

```diff
@@ -1,19 +1,25 @@
 # frozen_string_literal: true
 
+require 'digest'
+
 module SAML
   module UserAttributes
-    class SSOe < Base
+    class SSOe
```

Comment:
Could this class be a candidate for SimpleDelegator / or decorator pattern? It looks like you're running up against a CodeClimate refactoring warning. Just a thought.

Reply:
The reason I suggested this inheritance go away is that logically there is no structural relationship between the "Base" SAML attribute class (which represents the commonality inherent between the ID.me-issued assertions) and this SSOe class which represents an SSOe-issued assertion. 

There's a wee bit of shared logic primarily around the sign_in/account_type logic that maybe could be a mix-in or something but I think splitting apart this inheritance is the right call for clarity.